portfolio/src/main/webapp/script.js

```diff
@@ -1,3 +1,10 @@
 function changeImage() {
   document.getElementById("switch").className = "index-image-2"; 
-}
\ No newline at end of file
+}
+
+async function populateGreeting() {
```

Comment:
nit: add JSDoc comment to the function.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments